development/extensions/tutorial_notifications.rst

```diff
@@ -0,0 +1,955 @@
+=======================
+Tutorial: Notifications
+=======================
+
+.. contents:: This tutorial explains:
+   :depth: 1
+   :local:
+
+Introduction
+============
+Notifications have been a part of phpBB ever since :guilabel:`3.0`.
```

Comment:
I think you meant 3.1 here.

Reply:
3.0 already had email notifications if I am not mistaken. Atleast the `docs/CHANGELOG` suggest so. Only actual Board notifications came at a later time? I can change it to 3.1 anyway.

Reply:
2.0 also had email notifications (And if I remember well 1.4 as well) :). The notification system was introduced in 3.1 and that replaced the old email notifications stuff.

Reply:
Okay 👍 changed it to 3.1